routes/quotes.js

```diff
@@ -5,210 +5,48 @@ exports.register = function(server, options, next){
 
   server.route([
   
-    {
-      //GET all quotes by selecting author
-      method: 'GET',
-      path: '/quotes',
-      handler: function(request, reply){
-        reply(quotes);
-          var db = request.server.plugins['hapi-mongodb'].db;
-
-          db.collection('quotes').find().toArray({
-            function(err, quotes){
-             if (err) {return reply('Internal MongoDB error', err);}
-            }
-          })
-      }
-    },
-    //GET one quote, add select author or finding parts of that phrase from input box 
-    {
-      method: 'GET',
-      path: "/quotes/{id}",
-      handler:function(request, reply){
-        var quoteID = encodeURIcomponent(
-          request.params.id);
-        var db = request.server.plugin['hapi-mongodb'].db;
-        var ObjectId = request.server.plugins['hapi-mongodb'].ObjectID;
-
-        dbcollection('quotes').findOne({"_id": ObjectId(quoteID)}), function(err, quote){
-        if (err) {return reply('Internal MongoDB error', err);}
-
-        reply(quote);
-        }
-      }
-    },
-   
-   // POST a quote, would this be necessary?
-   {
-     method: 'POST',
-     path: '/quotes',
-     config:{
-       handler:function(request, reply){
-        Auth.authenticated(request,function(result){
-          if (result.authenticated) {
-            var db = request.server.plugins['hapi-mongodb'].db;
-
-            var tweet = {
-              "message": request.payload.quote.message,
-                "user_id: result.user_id"
-            };
-
-            db.collection('quotes').insert(quotes, function (err, writeResult){
-              if (err) {return reply('Internal MongoDB error', err);}
-
-              reply(writeResult);
-            });
-
-          }else {
-            reply(result.message);
+
+    //1. GET all quotes GET /quotes
+    method: 'GET',
+    path: '/quotes',
+    handler: function(request, reply){
+      reply(quotes);
```

Comment:
if you `reply(quotes)`, it doesn't matter what you write after. It will respond `undefined` and return